core/src/test/resources/WEB-INF/config-lucene.xml

```diff
@@ -125,6 +125,7 @@
       <field name="_valid_schematron-rules-iso" tagName="valid_schematron-rules-iso"/>
       <field name="_valid_schematron-rules-inspire" tagName="valid_schematron-rules-inspire"/>
       <field name="_valid_xsd" tagName="valid_xsd"/>
+      <field name="_valid_subtemplate" tagName="valid_subtemplate"/>
```

Comment:
If it has the same name for both record/template/subtemplate then red/green icon will work for all. If we introduce a new field, template for search results will have to be modified. Same for facet which will also work well on valid field for all kind of records ?